eth/filters/api.go

```diff
@@ -75,6 +75,11 @@ func NewPublicFilterAPI(backend Backend, lightMode bool) *PublicFilterAPI {
 	return api
 }
 
+// EventSystem returns
+func (filter *PublicFilterAPI) EventSystem() *EventSystem {
```

Comment:
This is unnecessary and also definitely wrong; you cannot expose internal structures like the event system through the API.